webapp/src/components/backstage/runs_list/row.tsx

```diff
@@ -184,3 +165,84 @@ const formatDate = (millis: number) => {
 };
 
 export default Row;
+
+const FollowPlaybookRun = ({id}: {id: string}) => {
+    const {formatMessage} = useIntl();
+    const currentUser = useSelector(getCurrentUser);
+    const [metadata] = useRunMetadata(id);
+    const [followers, setFollowers] = useState(metadata?.followers || []);
+    const [isFollowing, setIsFollowing] = useState(followers.includes(currentUser.id));
```

Comment:
```suggestion
    const isFollowing = followers.includes(currentUser.id);
```

I may be wrong, but this seems to be a derived property that don't need to be state